packages/notifications/tests/dummy/app/router.js

```diff
@@ -1,9 +1,9 @@
 import EmberRouter from '@ember/routing/router';
-import config from './config/environment';
+import config from 'dummy/config/environment';
```

Comment:
`navi-notifications/config/environment`?

Reply:
This came from the upgrade blueprint. It is referencing the dummy app's config file not the addons.

Reply:
👍